canonical-web-and-design/ubuntu.com
# Updated the page /openstack

## Done

- Updated the content of ubuntu.com/openstack based on the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#)

## QA

- Go to the website: 
- Make sure that the requested changes in the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#), and please approve the discussion in the [doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#) one the QA is done.

## Issue / Card

Fixes #10505
